test/solidity/Facets/DeBridgeFacet.t.sol

```diff
@@ -0,0 +1,242 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity 0.8.17;
+
+import { DSTest } from "ds-test/test.sol";
+import { DiamondTest, LiFiDiamond } from "../utils/DiamondTest.sol";
+import { Vm } from "forge-std/Vm.sol";
+import { DeBridgeFacet } from "lifi/Facets/DeBridgeFacet.sol";
+import { ILiFi } from "lifi/Interfaces/ILiFi.sol";
+import { LibSwap } from "lifi/Libraries/LibSwap.sol";
+import { LibAllowList } from "lifi/Libraries/LibAllowList.sol";
+import { ERC20 } from "solmate/tokens/ERC20.sol";
+import { UniswapV2Router02 } from "../utils/Interfaces.sol";
+import { IDeBridgeGate } from "lifi/Interfaces/IDeBridgeGate.sol";
+import "lifi/Errors/GenericErrors.sol";
+
+// Stub DeBridgeFacet Contract
+contract TestDeBridgeFacet is DeBridgeFacet {
+    constructor(IDeBridgeGate _deBridgeGate) DeBridgeFacet(_deBridgeGate) {}
+
+    function addDex(address _dex) external {
+        LibAllowList.addAllowedContract(_dex);
+    }
+
+    function setFunctionApprovalBySignature(bytes4 _signature) external {
+        LibAllowList.addAllowedSelector(_signature);
+    }
+}
+
+contract DeBridgeFacetTest is DSTest, DiamondTest {
```

Comment:
Could you maybe update your test file so that it inherits from TestbaseFacet.sol?
Please have a look at the test files of the other facets in order to understand how to do that. I am happy to support you in case you need any help

Reply:
updated it